rest_framework/views.py

```diff
@@ -280,6 +282,8 @@ def check_permissions(self, request):
         """
         for permission in self.get_permissions():
             if not permission.has_permission(request, self):
+                if hasattr(permission, 'message'):
+                    self.permission_denied(request, permission.message)
                 self.permission_denied(request)
```

Comment:
Could we simplify this to `self.permission_denied(request, message=getattr(permission, 'message'))`?